test/test_tree_2d_advection.jl

```diff
@@ -40,6 +40,21 @@ end
     end
 end
 
+@trixi_testset "elixir_advection_implicit_sparse_jacobian.jl with polydeg=2" begin
+    @test_trixi_include("/home/purple/trixi/Trixi.jl/examples/tree_2d_dgsem/elixir_advection_implicit_sparse_jacobian.jl",
+                        l2=[3.61524158e-03],
+                        linf=[6.44083371e-03],
+                        polydeg=2)
```

Comment:
We should probably stick to the original polynomial degree here, to make sure the example works indeed.